tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -317,7 +317,7 @@ def test_write_file(self):
 
     self.assertEqual(
       files_content_result.dependencies,
-      (FileContent("roland", b"European Burmese"),)
+      (FileContent("roland", b"European Burmese", False),)
```

Comment:
Encouraged to use kwarg `is_executable` here.

Reply:
Done